airbyte-integrations/bases/standard-test/standard_test/conftest.py

```diff
@@ -133,10 +133,10 @@ def pull_docker_image(standard_test_config) -> None:
 
 
 @pytest.fixture(name="expected_records")
-def expected_records_fixture(inputs, base_path) -> List[AirbyteMessage]:
+def expected_records_fixture(inputs, base_path) -> List[AirbyteRecordMessage]:
     path = getattr(inputs, "expected_records_path")
     if not path:
         return []
 
     with open(str(base_path / path)) as f:
-        return [AirbyteMessage.parse_raw(line) for line in f]
+        return [AirbyteRecordMessage.parse_raw(line) for line in f]
```

Comment:
why not make this `AirbyteMessage` in case we want to do some state as well?

Reply:
my logic here was the following:

- it is expected `records`
- STATE objects are kind of black boxes IIRC
- I don't think we should assert on LOG messages
- it is easier to provide records, ideally just `data` attribute, but we need to know the stream name